core/src/test/java/com/radixdlt/rev2/REv2ConsensusTransferTest.java

```diff
@@ -132,6 +133,7 @@ private static RawNotarizedTransaction createNewAccountTransaction() {
   }
 
   @Test
+  @Ignore("the TransactionBuilder's create_new_account_intent_bytes() helper is temporarily broken")
```

Comment:
I'd suggest creating a `createNewVirtualAccountWithXrdTransaction(publicKey)` via using:
```
var virtualAccountAddress = Address.virtualAccountAddress(publicKey);
...
```

And a manifest which transfers XRD to the virtual account, thereby creating it:
```
CALL_METHOD
    ComponentAddress("${virtualAccountAddress}") 
    "deposit_batch"
    Expression("ENTIRE_WORKTOP");
```

Reply:
The "virtual account" snippets are super helpful for adjusting many of the failing tests, thanks 👍 

However this one here is named `new_account_creates_transfer_of_xrd_to_account`, which suggests that it wants to test exactly the "new account with resource" function, which is no longer available (in fact, this is the only test that calls Rust jni `create_new_account_intent_bytes()` method).

If I adjust the `create_new_account_intent_bytes()` to use virtual account, then it would mean renaming it (to e.g. `deposit_xrd_to_virtual_account_intent_bytes()`) and also renaming the test to `can_transfer_xrd_to_account`. I could do it, but it is something already tested by other tests.

So I think we are left with options:
A) Keep the test `@Ignore`d temporarily, since Josh suggests that "scrypto side will get updated" (making this function available [and worth testing] again, I assume).
B) Remove the test and the entire jni `create_new_account_intent_bytes()`.

Reply:
I'm not sure (A) will be added back, and if it is, I don't think it will be tested here, so happy with (B).

That said, I think this might be the only test of the `get_component_resources` function - but that's just a function for use by tests anyway so I don't think it's a problem.

Reply:
if I am looking at the right `get_component_resources`, then it is still used by other tests (via `get_component_xrd`).

So with no remorse, I deleted this test 🤷‍♂️